.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+_Thanks for wanting to report an issue you've found in node.js. Please delete
+this text and fill in below template. If unsure about something, just do as best
+as you're able._
+
+_Note that it will be much easier for us to fix the issue if we will have a test
+case that reproduces the problem. Ideally this test case should not have any
+external dependencies. We understand that it is not always possible to reduce
+your code to a small test case, but we will appreciate to have as much data as
+possible. Thank you!_
+
+* **Version**: _enter affected node.js version(s)_
+* **Platform**: _either `uname -a` output, or if Windows version and 32-bit or
```

Comment:
I'm not how to write it down, but the idea is that 32-bit/64-bit question is related only to Windows.